blog/2023-06-05-discord-bot/index.md

```diff
@@ -0,0 +1,737 @@
+---
+slug: knowledge-base-discord-bot
+title: Build your own Discord/Slack bot using Supabase and OpenAI's GPT to help support teams
+authors: [fatonramadani]
+tags:
+  [
+    'Discord bot',
+		'Slack bot',
+    'Product documentation',
+    'Windmill',
+    'OpenAI',
+    'Supabase',
+    'Knowledge base',
+    'Natural language processing',
+    'AI-powered assistance',
+    'Developer tools',
+    'Documentation automation'
+  ]
+image: ./ai-bot.png
+---
+
+![AI BOT](./ai-bot.png 'AI BOT')
+_**Midjourney prompt:** a blog cover about a discord bot that answers questions about technical documentations, surrended by books --aspect 3:2_
+
+## Replace a SaaS
+
+This blog series cheeky title is to emphasize that you don't always need a dedicated SaaS to solve a problem. You can often build your own minimal but tailored to your needs solution on your own, and stop accumulating subscriptions.
+
+Today, we will create a bot that answers questions about technical documentation with a bot that uses Windmill, OpenAI, and Supabase with an approval step.
+
+## Demo
+
+<video
+	className="border-2 rounded-xl object-cover w-full h-full"
+	autoPlay
+	loop
+	controls
+	src="/videos/demo-bot.mp4"
+/>
+
```

Comment:
Video does not leave time to read what's happening, in particular:
- what is the prompt
- Faton approving the anwer